src/rollup-boost/rollup_boost_launcher.star

```diff
@@ -8,9 +8,9 @@ ethereum_package_constants = import_module(
 
 ENTRYPOINT_ARGS = ["/usr/local/bin/rollup-boost"]
 
-ROLLUP_BOOST_IMAGE_NAME="0x00101010/rollup-boost:latest"
+ROLLUP_BOOST_IMAGE_NAME="rollup-boost:latest"
```

Comment:
This is local image?

Reply:
Yeah, i feel like referencing local image is easier if want to rebuild a modified image